app/js/controllers/foyer/ressources/ressources.js

```diff
@@ -9,6 +9,11 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
     $scope.yearMoinsUn = moment($scope.situation.dateDeValeur).subtract('years', 1).format('YYYY');
     $scope.currentMonth = moment($scope.situation.dateDeValeur).format('MMMM YYYY');
 
+    // Pour les Auto-entrepreneurs
+    $scope.onGoingLabel = 'J\'aurai un chiffre d\'affaires non nul en ' + $scope.currentMonth;
```

Comment:
Ok… This is `"validateQuoteMarks": { "mark": "'", "escape": true }` in [JSCS](http://jscs.info/rules.html#validatequotemarks), but it turns out JShint doesn't support it.

Either ignore this comment or use `’` for the time being, then.